airflow/configuration.py

```diff
@@ -253,6 +253,18 @@ def _validate_config_dependencies(self):
                     + ", ".join(start_method_options)
                 )
 
+        if self.has_option("scheduler", "file_parsing_sort_mode"):
+            list_mode = self.get("scheduler", "file_parsing_sort_mode")
+            file_parser_modes = {"modified_time", "random_seeded_by_host", "alphabetical"}
+
+            if list_mode not in file_parser_modes:
+                raise AirflowConfigException(
+                    "`[scheduler] file_parsing_sort_mode` should not be "
+                    + list_mode
```

Comment:
```suggestion
                    + repr(list_mode)
```

Reply:
Use an f-string too maybe for this whole lote?

Reply:
fixed in https://github.com/apache/airflow/pull/15046/commits/e394febf64baffbeee34bc0383c042262659b424